### PR TITLE
Add documentation for Wireguard Threaded NAPI feature

### PR DESCRIPTION
**Pull request by jrcichra**
Product Version(s):

Issue:  https://github.com/projectcalico/calico/pull/9260

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico/network-policy/encrypt-cluster-pod-traffic.mdx [hunk 1]
```diff
@@ -216,6 +218,43 @@ The above command can be used to change other WireGuard attributes. For a list o
...
+Enable Threaded NAPI WireGuard across all the nodes using the following command.
+
+```bash
+kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardThreadingEnabled":true}}'
+```
+
+This enables Threaded NAPI for both the IPv4 and IPv6 WireGuard interfaces.
+
+</TabItem>
+<TabItem label="Manifest" value="Manifest-4">
```
Comment (mazdakn) [inline calico/network-policy/encrypt-cluster-pod-traffic.mdx]: I don't think we need a separate tab for Operator and manual install. Even in manual install it's possible to use `kubectl` to patch felixConfiguration.

**Comment by mazdakn** [comment]
@jrcichra Thanks for the documentation PR. You also need to make the same changes in the same file under `calico-enterprise` and `calico-cloud` directories.

**Comment by mazdakn** [comment]
Once the cherry picks and enterprise PRs are merged, we need to copy the same content to the corresponding directories.

**Comment by ctauchen** [comment]
We have about a dozen Wireguard-related Felix configuration parameters. Of those, only two are mentioned on this page, and both of them are required for the basic procedure to enable this feature.

So my first question: Why include this more obscure parameter so prominently?

Second: Would the warning be better placed (in a condensed form) in the parameter description in the configuration pages?

CC @jrcichra @mazdakn 

We've recently changed the way we produce the Felix configuration docs. Changes to parameter descriptions need to be made in the codebase at projectcalico/calico. Perhaps see @fasaxc for details.

**Comment by jrcichra** [comment]
I see. We can put this warning in the parameter description in the configuration pages instead, since it is an obscure feature like the rest of the tunables.

**Comment by ctauchen** [comment]
@jrcichra Sure thing -- if you're happy to do that, you can open up a PR in `projectcalico/calico`. I'll close this PR for now.

Again, if you need help finding the right spot, or questions about process, @fasaxc can sort you out.

**Comment by mazdakn** [comment]
@jrcichra You can probably add extra comments to the new config as comments. This way, the text will be populated in the docs automatically. Please tag me whenever you have the PR.

**Comment by jrcichra** [comment]
I've made a PR on the main project here: https://github.com/projectcalico/calico/pull/9440

**Pull request closed without merging**